src/openbmp/api/parsed/message/Base.py

```diff
@@ -76,20 +100,28 @@ def parse(self, version, data):
         if len(self.header_names) == 0:
             raise Exception("header_names should be overriden.")
 
-        records = data.splitlines()
-
-        # # Splits each record into fields.
-        for r in records:
-            fields = r.split('\t')  # Fields of a record as array.
-
-            # # # Process and validate each field with its corresponding processor.
-            if len(fields) >= len(self.processors):
-                for i,processor in enumerate(self.processors):
-                    fields[i] = processor.process_value(fields[i])
-
-            fields_dict = dict(zip(self.header_names, fields))
-
-            self.row_map.append(fields_dict)
+        # Splits each record into fields.
+        for record in Base.isplit(data, "\n"):
+            fields = record.split('\t')  # Fields of a record as array.
+
+            fields_map = {}
```

Comment:
Looks like your conflict merge reverted changes that should be here. Please check your merge conflict changes and correct here.